.teamcity/src/main/kotlin/configurations/PerformanceTestsPass.kt

```diff
@@ -35,6 +36,13 @@ class PerformanceTestsPass(model: CIBuildModel, performanceTestProject: Performa
 
         applyDefaultSettings(os)
         params {
+            text(
+                "reverse.dep.*.performance.baselines",
```

Comment:
It seems like the scope of this is too large, though I am not sure. We don't have performance tests depending on other performance tests, do we? Can we somehow use `additional.gradle.parameters` for this use-case?

Reply:
The problem is that we want to set the parameter in the trigger build and let the trigger build pass the parameter to all individual performance test, so `additional.gradle.parameters` doesn't work here, we have to use `reverse.dep.*.additional.gradle.parameters`, which is no difference from `reverse.dep.*.performance.baselines`.